be1-go/hub/organizer.go

```diff
@@ -142,15 +151,20 @@ func (o *organizerHub) handleMessageFromClient(incomingMessage *IncomingMessage)
 		// Verify the data
 		err := o.verifyJson(msg.RawData, DataSchema)
 		if err != nil {
-			log.Printf("failed to validate the data: %v", err)
+			err = message.NewError("failed to validate the data", err)
 			client.SendError(&id, err)
 			return
 		}
 
 		// Unmarshal the data
 		err = query.Publish.Params.Message.VerifyAndUnmarshalData()
 		if err != nil {
-			log.Printf("failed to verify and unmarshal data: %v", err)
+			// Return a error of type "-4 request data is invalid" for all the verifications and unmarshalling problems of the data
+			err = &message.Error{
+				Code:        -4,
+				Description: fmt.Sprintf("failed to verify and unmarshal data: %v", err),
+			}
+			log.Printf("%v", err)
```

Comment:
This can be removed since there's a log in `client.SendError`

Reply:
Oh yes that is correct!